addons/gamepad/gamepadlib.js

```diff
@@ -0,0 +1,835 @@
+let console = window.console;
```

Comment:
should this be moved to the `libraries` folder?

Reply:
no